packages/node_modules/@webex/webex-core/test/unit/spec/interceptors/embargo.js

```diff
@@ -0,0 +1,139 @@
+import chai from 'chai';
```

Comment:
This is an enhanced test that covers additional ground. Reminder, that interceptors can only be tested as a unit tests since they are loaded into non-addressable namespaces in an instance function.